### PR TITLE
rktnetes documentation improvements

### PR DESCRIPTION
**Pull request by yifan-gu**
Based on https://github.com/kubernetes/kubernetes.github.io/pull/672/files

cc @kubernetes/sig-rktnetes 

We need this to be cherry-picked into 1.3 release as well.

### CODE REVIEW

**Comment by yifan-gu** [comment]
@googlebot Both @euank and @yifan-gu should have signed under coreos.com

**Comment by yifan-gu** [comment]
cc @s-urbaniak

**Comment by erictune** [comment]
@yifan-gu are all the statements in here true about 1.2 as well as 1.3?  If not, maybe this should only be a PR against the 1.3 branch?

**Comment by yifan-gu** [comment]
@erictune They are true about 1.3. 
OK I will close this and reopen agains 1.3, thanks for reminding :)

File: docs/getting-started-guides/rkt/index.md [hunk 1]
```diff
@@ -135,52 +130,26 @@ Then you can launch the cluster by:
...
 
 See [a simple nginx example](/docs/user-guide/simple-nginx) to try out your new cluster.
 
 For more complete applications, please look in the [examples directory](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/).
 
-### Different UX with rkt container runtime
+### Known Issues and Differences
 
-rkt and Docker have very different designs, as well as ACI and Docker image format. Users might experience some different experience when switching from one to the other. More information can be found [here](/docs/getting-started-guides/rkt/notes/).
+rkt and Docker have very different designs, as well as ACI and Docker image format.
```
Comment (tmrts) [inline docs/getting-started-guides/rkt/index.md]: Mention OCI image spec maybe?

File: docs/getting-started-guides/rkt/notes.md [hunk 2]
```diff
@@ -1,99 +1,91 @@
 ---
 ---
 
-# Notes on Different UX with rkt container runtime
+# Known issues
 
-### Doesn't support ENTRYPOINT + CMD feature
+The following features all either do not operate or have large caveats when using the rkt container runtime.
```
Comment (tmrts) [inline docs/getting-started-guides/rkt/notes.md]: nit: s/all either do not operate/either are not supported/

**Comment by tmrts** [comment]
@yifan-gu LGTM, make sure we're okay with the 1.9 version and feel free to apply LGTM and merge it

**Pull request closed without merging**